src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -13273,159 +13273,22 @@ Map {
       },
     },
   },
-  "Accordion" => Object {
```

Comment:
Not quite sure what these changes are. Will have to check what's happening here.

Reply:
OK, these were not removed just moved down the file. It is good to go.